users/tests/factories.py

```diff
@@ -0,0 +1,39 @@
+import random
+
+from factory import DjangoModelFactory, Faker, SubFactory
+from factory.django import FileField
+
+from users.models import User, UsersHaveMadeOut, KSG_STATUS_TYPES, KSG_ROLES
+
+
+class UserFactory(DjangoModelFactory):
+    class Meta:
+        model = User
+
+    # From AbstractUser model
+    username = Faker('user_name')
```

Comment:
Should this (and similar fields) perhaps be changed to [Sequences](https://factoryboy.readthedocs.io/en/latest/introduction.html?highlight=Sequence#sequences)?

Reply:
Agreed!

Reply:
I couldn't make these changes to `InternalGroupFactory`, since the unique name property should be chosen from the KSG_INTERNAL_GROUPS tuple